cmd/blockchaincmd/deploy.go

```diff
@@ -925,7 +925,9 @@ func deployBlockchain(cmd *cobra.Command, args []string) error {
 			return err
 		}
 		ux.Logger.PrintToUser("")
-
+		if err = setBootstrapValidatorValidationID(avaGoBootstrapValidators, bootstrapValidators, subnetID); err != nil {
```

Comment:
question: do we need to do the same after we add validator? 
also it's possible if the same validator will be removed and added back.